code/lib/core-server/src/utils/server-statics.ts

```diff
@@ -41,10 +41,7 @@ export async function useStatics(router: any, options: Options) {
                 directory: dir,
               })
             : dir;
-          const { staticDir, staticPath, targetEndpoint } = await parseStaticDir(relativeDir);
-          logger.info(
-            chalk`=> Serving static files from {cyan ${staticDir}} at {cyan ${targetEndpoint}}`
```

Comment:
I always thought this was quite useful, especially as a way to debug issues. If you feel really strong about removing them, can we at least change it to logger.debug?

Reply:
I agree this is useful, as does the original commenter

Reply:
OK, well if others find it useful, I'll add it back with a filter..

Doe y'all like this solution better?